docs/contributing.rst

```diff
@@ -94,3 +94,28 @@ To initialize the pre-commit hook script to run in your development branch, run
 
     $ rm .git/hooks/pre-commit
 
+
+Release Branch Maintenance
+==========================
+
+Fix PRs
+-------
+
+Fix PRs are prefixed with fix and are generally prompted by a bug reported by a user of the library. For this kind of PRs, the backporting decision should work as follows.
+The user reports a bug against the X release. If X is no longer maintained, ask the user to upgrade to a maintained version Y. The fix should then be backported down to release Y, if the bug is not already fixed there. Otherwise, if X is still maintained, it should be backported down to release X.
```

Comment:
This wording might make the most sense internally. Perhaps we should consider rewriting this considering a wider audience (albeit I don't have a concrete suggestion in mind to contribute at the moment 🙁 )?